ph-ee-connector-ams-mifos/src/main/java/org/mifos/connector/ams/interop/AmsCommonService.java

```diff
@@ -129,6 +143,36 @@ public void repayLoan(Exchange e) {
         // cxfrsUtil.sendInOut("cxfrs:bean:ams.local.loan", e, headers, e.getIn().getBody());
     }
 
+    public String disburseLoan(String fineractTenantId, LoanDisbursementRequestDto loanDisbursementRequestDto, String channelRequest,
```

Comment:
method  name is not accurately reflecting its functionality, it could be processLoanDisbursementRequest

Reply:
there is already a function for loan repayment and it's name is repayLoan. I believe disburseLoan is an appropriate name it we look how functions are named in the ams common service. It you still want to change it, let me know.